### ISSUE TITLE
Odd bug

### ISSUE THREAD

**Issue by driva5** [ChaosinaCan/ClassicTabs#32]
Firstly, I use your add-on to **activate the next tab when closing a tab**. Playing around with your add-on for the past day, I have noticed this unusual bug.

My experience:

Opening two links in two new tabs from a single website source will cause the add-on to malfunction. 

This is difficult to explain but lets pretend this is my tab bar, I have 4 tabs open:

1.O | 2.N | 3.N | 4.R

O = Original Tab
N = New Tab created from original tab
R = Random Tab

Selecting Tab 2 (N) and closing it, Opera will correctly switch to Tab 3 (N). 
When Tab 3 (N) becomes the activate tab and I close it, Opera will incorrectly switch to the previous tab, Tab 1 (O). It should switch to Tab 4 (R) as that's the next tab.

Open this link for a video demonstration:
https://streamable.com/glz08

I then discovered that this behaviour only occurs when the amount of tabs created is an even number. If the amount of tabs created is an odd number, Opera will correctly switch to the next tab.

Open this link for a video demonstration:

https://streamable.com/8tw74

**Comment by ChaosinaCan** [comment]
I've sort of given up on making that feature work correctly now. I can't tell Opera which tab to focus after closing a tab, so instead I watch for when Opera focuses a different tab immediately after closing the one you were on, and then quickly tell it to focus a different tab. It seems to be really sensitive to changes in timing, and Opera finds a new way to break it every few versions.

I'm not sure if it's been fixed yet, but there's been a bug in Opera for a while where this extension could swap where two tabs appear to be in the tab bar, and upon interacting with them again, they'd jump back where they're supposed to be.

**Comment by driva5** [comment]
That's unfortunate. The options that your add-on provide should be available natively in Opera. Appears older versions of Opera had these options but were removed for whatever stupid reason. Seems like the upkeep of your add-on is a pain in the ass, so I don't blame you for not wanting to update it. Hopefully the developers at Opera come to their senses and reintroduce these useful options.

**Comment by richarson** [comment]
@ChaosinaCan yeah, that bug still exists. I think it only happens when you interact with the last two/three tabs on the right, and not always either.

As with many things, I've learnt to live with it :)
